apps/package.json

```diff
@@ -27,7 +27,10 @@
   },
   "browserify-global-shim": {
     "react": "React",
-    "react-dom": "ReactDOM"
+    "react-dom": "ReactDOM",
+    "johnny-five": "JohnnyFive",
+    "playground-io": "PlaygroundIO",
+    "chrome-serialport": "ChromeSerialport"
```

Comment:
You can see an example in https://github.com/code-dot-org/code-dot-org/pull/7583 where I do this for marked.